packages/dx-react-grid/docs/reference/table-header-row.md

```diff
@@ -4,7 +4,7 @@ A plugin that renders the table's header row. The [Column](grid.md#column)'s `ti
 
 The plugin also allows you to manage a column's sorting and grouping state and initiate column dragging.
 
-## Importing
+## Import
```

Comment:
@viterobk We should change it it all references. Please revert it

Reply:
I changed all the occurrences (https://github.com/DevExpress/devextreme-reactive/pull/1358)